lib/mixins/template-stamp.html

```diff
@@ -313,6 +315,32 @@
        *   metadata to `nodeInfo`
        */
       static _parseTemplateNestedTemplate(node, outerTemplateInfo, nodeInfo) {
+        // If a `<dom-repeat>` or a `<dom-if>` is inside of an svg, we need
+        // to create all elements in a different namespace. To do so, make an svg,
+        // let the HTML parser parse in that namespace, and then put back all
+        // created children in a fresh template.
+        if (node.parentNode.localName && templateExtensions[node.parentNode.localName]
+              && node.parentNode.parentNode && node.parentNode.parentNode.namespaceURI === SVG_NAMESPACE) {
+          const svg = document.createElementNS(SVG_NAMESPACE, 'svg');
+          svg.innerHTML = node.innerHTML;
+
+          const template = /** @type {HTMLTemplateElement} */ (document.createElement('template'));
+
+          while (svg.firstChild) {
+            template.content.appendChild(svg.removeChild(svg.firstChild));
```

Comment:
Tip: no need to remove a child when moving it; append to new spot removes it from the previous, since a node can't exist in two trees at the same time.